File: 80-remove-duplicates-from-sorted-array-ii/80-remove-duplicates-from-sorted-array-ii.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        j, count = 1,1
        for i in range(1,len(nums)):
            if nums[i-1] == nums[i]:
                count += 1
            else:
                count = 1
            if count <= 2:
                nums[j] = nums[i]
                j += 1
        return j

File: 80-remove-duplicates-from-sorted-array-ii/80-remove-duplicates-from-sorted-array-ii.js
/**
 * @param {number[]} nums
 * @return {number}
 */

var removeDuplicates = function(nums) {
    let count = 0, k = 0;

    for(let i = 0; i < nums.length; i++) {
        if(nums[i] !== nums[i+1]) {
            nums[k] = nums[i];
            k++;
            count = 0;
        }
        if (nums[i] === nums[i+1] && count < 1) {
            nums[k] = nums[i];
            k++;
            count++;
        }
    }
    return k;
};